src/server/game/Battlegrounds/Battleground.h

```diff
@@ -234,6 +234,7 @@ enum BGHonorMode
 
 #define BG_AWARD_ARENA_POINTS_MIN_LEVEL 71
 #define ARENA_TIMELIMIT_POINTS_LOSS    -16
+#define RATED_BATTLEGROUND_STATE_ACTIVE 5508
```

Comment:
Put WS_ behind that enum to clarify that it's a world state to follow the remaining world state enum code style